src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final int INDEX_OF_AMOUNT = 1;
+    public static final int INDEX_OF_OPERATION_TYPE = 0;
+    public static final String ELEMENT_TO_SEPARATE = " ";
+    public static final String SYMBOL_COMA = ",";
+    public static final String OPERATOR_TYPE_SUPPLY = "supply";
+    public static final String OPERATOR_TYPE_BUY = "buy";
+    public static final String OPERATOR_TYPE_RESULT = "result";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String [] dataArray = readFromFile(fromFileName);
+        String report = createReport(dataArray);
+        writeToFile(toFileName, report);
+    }
+
+    public String[] readFromFile(String fromFileName) {
+        File file = new File(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(ELEMENT_TO_SEPARATE);
```

Comment:
What if we add a `comment` column and some cells contain `We need to buy more apples`?